execution/executor-types/src/lib.rs

```diff
@@ -256,7 +258,7 @@ impl ExecutedTrees {
     pub fn version(&self) -> Option<Version> {
         let num_elements = self.txn_accumulator().num_leaves() as u64;
         if num_elements > 0 {
-            Some(num_elements - 1)
+            Some(num_elements.saturating_sub(1))
```

Comment:
@msmouse  :D?

Reply:
I think this should be fine as saturating_sub will not wrap around since  num_elements > 0

Reply:
ah yeah, so a better way to write that is to remove the entire `if-else` and replace it with:

```rust
num_elements.checked_sub(1)
```

this will return either `Some(result)` or `None` if `num_elements == 0`